.travis.yml

```diff
@@ -117,3 +122,14 @@ jobs:
       before_script: skip
       script:
         - beaver bintray upload -D $DIST_VERSION -N $CACHE_DIR/*.deb
+    - <<: *deploy
+      # Build code coverage for upstream minor branches and upload to coveralls.io
+      stage: deploy
+      if: branch =~ ^v[0-9]+\.[0-9]+\.x$ AND
+          NOT type IN (pull_request) AND
+          repo = sociomantic-tsunami/libdrizzle-redux
+      env:
+          MAKE_TARGET="$MAKE_TARGET:code-coverage-capture" CONFIGURE_ARGS=--enable-code-coverage
+      before_script: skip
+      script:
+        - ./travis_configure.sh upload_coverage "$CACHE_DIR/libdrizzle-redux-coverage.info"
```

Comment:
Uploading coverage reports doesn't seem really like "deployment", but I guess is not important.

Reply:
True. It isn't an artifact as such. However it should depend on the success of builds in the `test` stage. Adding yet another stage only for coverage upload would also complicate the config.